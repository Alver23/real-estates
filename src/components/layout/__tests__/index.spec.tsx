// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import LayoutApp from '../index';

describe('<LayoutApp />', () => {
  let component: ShallowWrapper;

  const children = <p>Children content</p>;
  const props = {
    children,
  };

  beforeEach(() => {
    component = shallow(<LayoutApp {...props} />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render correctly with chldren', () => {
    expect(component.find('p').exists()).toBeTruthy();
  });
});
