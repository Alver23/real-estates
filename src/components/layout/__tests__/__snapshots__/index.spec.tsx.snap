// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should save a snapshot of the component 1`] = `
<div
  className="bg-white"
>
  <Header />
  <div>
    <p>
      Children content
    </p>
  </div>
  <Footer />
</div>
`;
